### PR TITLE
GHMD-245

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/components/dashboard/Hub/Hub.tsx [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
-import { HubCardProps } from './Card'
+import Card, { HubCardProps } from './Card'
+import { useLocation } from 'react-router-dom'
 
 type Props = {
   cards: HubCardProps[]
 }
 
 function Hub({ cards }: Props) {
+  const [showEnrollment, setShowEnrollment] = useState(false)
```
Comment (jay-hodgson) [inline src/components/dashboard/Hub/Hub.tsx:12]: default is showEnrollment=false

**Comment by jay-hodgson** [comment]
Merging to get this out to staging for Solly soon, but will follow up with any recommended code changes.

File: src/components/dashboard/Hub/Hub.tsx [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+  const location = useLocation()
+  const params = new URLSearchParams(location.search)
+  const forceShowEnrollmentURLParam = params.get('forceShowEnrollment')
+
+  useEffect(() => {
```
Comment (conundrumgirl) [inline src/components/dashboard/Hub/Hub.tsx:17]: just wondering if we need the useEffect or even have `showEnrollment` in the state.  From the code it appears that the  only time forceShowEnrollmentURLParam gets set is when the components loads. If this is the case, won't 
` const showEnrollment = params.get('forceShowEnrollment') === 'true'` be sufficient?

Comment (jay-hodgson) [reply src/components/dashboard/Hub/Hub.tsx:17]: I think you're right!  Let me test one thing (updating the URL to the same route but different query params to verify that it's reloaded)...

Comment (jay-hodgson) [reply src/components/dashboard/Hub/Hub.tsx:17]: It sure does.  Thanks @conundrumgirl , I'll update the code for future releases...

**Pull request merged**